<launch>
	<!-- global rosparam -->
	<param name="use_sim_time" value="true"/>

	<!-- rosbag -->
	<node pkg="rosbag" type="play" name="player" args="--clock $(env HOME)/rosbag/infant/dkan_outdoor/2019-01-13-15-46-58.bag -s 60"/>

	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find pc_normal_estimation)/rviz_config/velodyne.rviz"/>

	<!-- sensor driver -->
	<include file="$(find velodyne_pointcloud)/launch/32e_points.launch"/>

	<!-- main -->
	<node pkg="pc_normal_estimation" name="normal_estimation" type="normal_estimation" output="screen">
		<remap from="/point_cloud" to="/velodyne_points"/>
		<param name="search_radius" type="double" value="0.5"/>
	</node>
</launch>
